app/src/main/assets/GimelGimel/GGBootstrap.js

```diff
@@ -35,16 +37,18 @@ GG.Utils.setScreenSpaceEventAction(GG.viewer, Cesium.ScreenSpaceEventType.LEFT_D
         var longitude = Cesium.Math.toDegrees(cartographic.longitude);
         var latitude = Cesium.Math.toDegrees(cartographic.latitude);
 
-        GG.AndroidAPI.updateLocation({
+        GG.AndroidAPI.updateSelectedLocation({
             longitude: longitude,
             latitude: latitude
         });
     }
 });
 
-GG.layerManager = new GG.Layers.LayersManager(GG.viewer);
-GG.cameraManager = new GG.CameraManager(GG.viewer.camera);
+GG.Utils.setScreenSpaceEventAction(GG.viewer, Cesium.ScreenSpaceEventType.LEFT_UP, function(movement) {
```

Comment:
There's a later comment explaining why using LEFT_UP screen space event causes us problems.
Cesium exposes camera movement events:
viewer.camera.moveEnd.addEventListener
I'd be better to use that so that ALL camera movement events will trigger the camera position update instead of only camera movement events caused by left clicking !resolve